@types/index.d.ts

```diff
@@ -3,10 +3,16 @@
 /* eslint-disable no-var, import/no-mutable-exports */
 
 import {Agent} from 'http';
-import {AbortSignal} from 'abort-controller';
-import Blob from 'fetch-blob';
+import * as Blob from 'fetch-blob';
 
-type HeadersInit = Headers | string[][] | Record<string, string>;
+export type AbortSignal = {
+    readonly aborted: boolean;
+
+    addEventListener(type: "abort", listener: (this: AbortSignal, ev: Event) => any, options?: boolean | { passive?: boolean; once?: boolean; }): void;
+    removeEventListener(type: "abort", listener: (this: AbortSignal, ev: Event) => any, options?: boolean | { capture?: boolean; }): void;
+};
+
+export type HeadersInit = Headers | string[][] | Record<string, string>;
```

Comment:
If you want to export `HeadersInit` it must be described fully and correctly. Right now it doesn't cover all possible ways to construct `Headers`, particularly from `Iterables`. Look how it's described in JSDoc at `headers.js`:

```suggestion
export type HeadersInit = Headers | Record<string, string> | Iterable<readonly [string, string]> | Iterable<string>[];
```

Reply:
But still I'm 👎  to export `HeadersInit`. What is use case for this?

```ts
const hi: HeadersInit = { /*...*/ }
const header = new Headers(hi)
```
?

That looks weird.

Reply:
Imagine a library which do this:
```typescript
function myCustomHttpGet(url: string, headers: HeadersInit): Promise<void>
```

It's important for this library to be able to access all types for the underlying library to avoid duplication and reduce maintenance.